app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,44 @@
+class RecipesController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @recipes = current_user.recipes
+  end
+
+  def show
+    @recipe = Recipe.find(params[:id])
+    @recipe_food = RecipeFood.new
+  end
+
+  def new
+    @recipe = Recipe.new
+  end
+
+  def create
+    recipe = Recipe.new(recipe_params)
+    recipe.user = current_user
+
+    if recipe.save
+      redirect_to recipe, notice: 'Recipe was successfully created.'
+    else
+      redirect_to new_recipe_path, notice: 'There was an error creating the recipe.'
+    end
+  end
```

Comment:
When trying to create recipes, I was unable to do so as it was throwing this error on the screen. ⬇️
![image](https://user-images.githubusercontent.com/33268927/202848368-d6688f4e-c939-42a6-90eb-2d505b35c926.png)
You can also watch [this video](https://www.loom.com/share/09be395ccc8744d986d288554cebd7f5) where I demonstrated what exactly is happening.